elements/rh-stat/rh-stat.css

```diff
@@ -4,9 +4,9 @@
 
 :host {
     display: flex;
-    --accent-color: var(--rh-footer-accent-color, #e00);
-    --red-hat-text:  var(--rh-theme--font-family--heading, 'Red Hat Text');
-    --red-hat-text-display: var(--rh-theme--font-family--heading, 'Red Hat Display');
+    --accent-color: var(--rh-theme--color--ui-accent, #e00);
```

Comment:
nit: `style` commit type is actually for _code style_ i.e. refactorings, lintings, things like that.

CSS changes should have `fix:` commit type. Confusing? yeah, just one of those things

Reply:
Got it, that makes sense. I was curious why style was mentioning white-space but didn't seem like the other categories really fit either.  Will keep in mind for next time.